infra/scripts/test-end-to-end.sh

```diff
@@ -5,11 +5,13 @@ set -o pipefail
 [[ $1 == "True" ]] && ENABLE_AUTH="true" || ENABLE_AUTH="false"
 echo "Authenication enabled : ${ENABLE_AUTH}"
 
-test -z ${GOOGLE_APPLICATION_CREDENTIALS} && GOOGLE_APPLICATION_CREDENTIALS="/etc/gcloud/service-account.json"
+test -z ${GOOGLE_APPLICATION_CREDENTIALS} && GOOGLE_APPLICATION_CREDENTIALS=""
```

Comment:
Why are we clearing that configuration? Isn't that a default?

Reply:
`install_gcloud_sdk()` tries to activate service account credential if this variable is set. Under the current `.prow/config.yaml` `test-end-to-end` does not expose the service account key.json.

Reply:
Reverted.